src/coq_rules.mli

```diff
@@ -0,0 +1,15 @@
+(* This file is licensed under The MIT License *)
+(* (c) MINES ParisTech 2018-2019               *)
+(*     Written by: Emilio Jesús Gallego Arias  *)
+
+(* Build rules for Coq's .v -> .vo files       *)
+
+open! Stdune
+
+val setup_rules
+  :  sctx:Super_context.t
+  -> dir:Path.t
+  -> dir_contents:Dir_contents.t
+  -> scope:Scope.t
```

Comment:
I would suggest getting rid of this argument if it's just for debugging. If you need the scope, then perhaps you should just fetch it using the `~dir` argument. Otherwise it's a bit confusing.

Reply:
Fixed thanks!